src/server/game/Entities/Object/Object.h

```diff
@@ -974,6 +975,8 @@ class WorldObject : public Object, public WorldLocation
         void GetGameObjectListWithEntryInGrid(std::list<GameObject*>& lList, uint32 uiEntry, float fMaxSearchRange) const;
         void GetCreatureListWithEntryInGrid(std::list<Creature*>& lList, uint32 uiEntry, float fMaxSearchRange) const;
 
+        void GetPlayerListInGrid(std::list<Player*>& lList, float fMaxSearchRange) const;
```

Comment:
```suggestion
        void GetPlayerListInGrid(std::list<Player*>& lList, float fMaxSearchRange) const;
```
Personally I'd to this to group them with the other InGrid stuff :shrug: